clients/drcachesim/tests/invariant_checker_test.cpp

```diff
@@ -2952,6 +2952,163 @@ check_exit_found(void)
     return true;
```

Comment:
> However, it should really expect the kernel_event marker to hold the actual return point of the signal

But there is not always a signal (used for rseq abort handler, APC, exception, callback too), and it might not return.  Better to describe as the interrupted PC: the one to re-execute if the event handler wants to go back.

Reply:
Done.

Reply:
Done.